src/screens/settings/Settings.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  Dimensions,
+} from 'react-native';
+import {useTranslation} from 'react-i18next';
+
+const {width} = Dimensions.get('window');
+
+const LANGS = [
+  {lngCode: 'en', label: 'English'},
+  {lngCode: 'hi', label: 'हिन्दी'},
+];
+
+export const Settings = () => {
+  const {t, i18n} = useTranslation();
+  const selectedLngCode = i18n.language;
+  const setLng = (lngCode: string) => i18n.changeLanguage(lngCode);
+  return (
+    <View style={styles.container}>
+      <View style={styles.titleContainer}>
+        <Text style={styles.select}>{t('languageSelector:selectLng')}</Text>
+      </View>
+      {LANGS.map((l) => {
+        const selected = l.lngCode === selectedLngCode;
+        return (
+          <TouchableOpacity
+            onPress={() => setLng(l.lngCode)}
+            key={l.lngCode}
+            disabled={selected}>
+            <View style={[styles.row, selected ? styles.selectedRow : {}]}>
+              <Text style={[selected ? styles.selectedText : styles.text]}>
+                {l.label}
+              </Text>
+              {selected && <Text>👍</Text>}
+            </View>
+          </TouchableOpacity>
+        );
+      })}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
please put this styles in styles file for the settings screen

Reply:
resolved